assets/app/scripts/controllers/projects.js

```diff
@@ -24,6 +24,45 @@ angular.module('openshiftConsole')
       $('#openshift-logo').off('click.projectsPage');
     });
 
+    $scope.toggleDelete = function(project) {
```

Comment:
whole project as a param is fine, but grab `var name = project.metadata.name;` as a local var at the beginning